### PR TITLE
Fixed a typo

### PR DESCRIPTION
**Pull request by AXE02**
Fixes #(https://github.com/GoogleChrome/developer.chrome.com/issues/877)
Changes proposed in this pull request:

-Fixed a simple typo

### CODE REVIEW

**Review by jpmedley — APPROVED** [review]

**Comment by jpmedley** [comment]
@AXE02 Can you please sign the CLA? There are instructions above.

**Review by awfuchs — APPROVED** [review]

**Comment by AXE02** [comment]
Hey.
I have signed it as instructed.

File: site/en/docs/android/custom-tabs/overview/index.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ any UI at all. It is recommended for developers who want to open their own
 ## Where is Custom Tabs available?
 
 Custom Tabs is a feature supported by browsers on the Android platform. It was originally
-introduced by [Chrome][2], on version 45. Currentely, the protocol is supported by most Android
```
Comment (tenjua) [inline site/en/docs/android/custom-tabs/overview/index.md:87]: N y p

Comment (tenjua) [inline site/en/docs/android/custom-tabs/overview/index.md:87]: N y p

Comment (tenjua) [inline site/en/docs/android/custom-tabs/overview/index.md:87]: N y P

Comment (tenjua) [inline site/en/docs/android/custom-tabs/overview/index.md:87]: N Y P

Comment (tenjua) [inline site/en/docs/android/custom-tabs/overview/index.md:87]: N Y P

Comment (tenjua) [inline site/en/docs/android/custom-tabs/overview/index.md:87]: N Y P

Comment (tenjua) [inline site/en/docs/android/custom-tabs/overview/index.md:87]: N Y P

**Review by tenjua — APPROVED** [review]

File: site/en/docs/android/custom-tabs/overview/index.md (hunk 1 above)
Comment (tenjua) [inline site/en/docs/android/custom-tabs/overview/index.md:87]: N Y P

Comment (tenjua) [inline site/en/docs/android/custom-tabs/overview/index.md:87]: N Y P

Comment (tenjua) [inline site/en/docs/android/custom-tabs/overview/index.md:87]: N Y P

**Review by tenjua — COMMENTED** [review]
N Y P

File: site/en/docs/android/custom-tabs/overview/index.md (hunk 1 above)
Comment (tenjua) [inline site/en/docs/android/custom-tabs/overview/index.md:87]: N Y P

**Review by tenjua — APPROVED** [review]

**Comment by jpmedley** [comment]
@AXE02, can you please sign the CLA. We can't merge this until you do.

**Comment by dotproto** [comment]
Removing @tenjua's comments as they do not meaningfully contribute to the PR review.

**Comment by dotproto** [comment]
@AXE02, this PR has been blocked because the [google-cla bot asked](https://github.com/GoogleChrome/developer.chrome.com/pull/883#issuecomment-860041735) you to reply to the issue with `@googlebot I signed it!`. Normally this is what triggers a rescan and unblocks the "cla/google" check that prevented us from merging the request. I have manually initiated the CLA signature check, so we are now unblocked.

**Pull request merged**